src/core/linalg/src/sparse/4C_linalg_fevector.cpp

```diff
@@ -37,161 +31,156 @@ Core::LinAlg::FEVector<T>::FEVector(const Epetra_FEVector& Source)
 {
 }
 
-
-
 template <typename T>
 Core::LinAlg::FEVector<T>::FEVector(const FEVector& other)
     : vector_(Utils::make_owner<Epetra_FEVector>(other.get_ref_of_epetra_fevector()))
 {
 }
 
-
 template <typename T>
 Core::LinAlg::FEVector<T>& Core::LinAlg::FEVector<T>::operator=(const FEVector& other)
 {
   *vector_ = other.get_ref_of_epetra_fevector();
   return *this;
 }
 
-
```

Comment:
I think we should add a guideline on how we visually separate function bodies in implementation files. Some people insert comment lines `/*---*/`, others use blank lines, others don't use anything. Changing this back and forth is noisy and distracts from the actual changes.

Reply:
I agree. The changes made in this PR just makes it consistent throughout the file to make it appear at least a bit tidy.